.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,56 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'bug'
+assignees: ''
+
+---
+
+### Describe the bug
+A clear and concise description of what the bug is.
+
+### Reproduction
+
+Provide a self-contained, concise snippet of code that can be used to reproduce the issue.
+For more complex issues provide a repo with the smallest sample that reproduces the bug.
+
+Avoid including business logic or unrelated code, it makes diagnosis more difficult.
+The code sample should be an SSCCE. See http://sscce.org/ for details. In short, please provide a code sample that we can copy/paste, run and reproduce.
+
+### Expected behavior
+A clear and concise description of what you expected to happen.
+
+---
+
+#### Datadog SDK version:
+
+_Which version of the Datadog SDK causes this problem? e.g. `1.2.0`_
```

Comment:
maybe worth setting `2` as a major version for both examples (one less char to edit)

Reply:
done